.travis/build-macos.sh

```diff
@@ -4,6 +4,68 @@ if echo $TRAVIS_TAG | grep ^v; then BUILD_TYPE=RelWithDebInfo; else BUILD_TYPE=D
 
 mkdir build
 cd build
-cmake -DCMAKE_OSX_DEPLOYMENT_TARGET=10.7 -DCMAKE_BUILD_TYPE=$BUILD_TYPE ..
-make VERBOSE=1
+cmake -DCMAKE_OSX_DEPLOYMENT_TARGET=10.9 -DCMAKE_BUILD_TYPE=$BUILD_TYPE ..
+make -j 2 VERBOSE=1
 make test_solvespace
+
+ls bin
```

Comment:
Can you split this into a different file, e.g. `.travis/sign-macos.sh`?

Reply:
Allright, will do!